release-process-documentation/documentation-guides/update-release-docs-new.md

```diff
@@ -1,88 +1,180 @@
-*   Merge `master` branch back into the branch for the last release. 
-For example, if you are currently on the v1.8 release, you will need to periodically merge `master` into `release-1.7` right up until `release-1.8` is merged into `master`. This is to make sure that the last release branch is as up to date as possible and left as a snapshot when the new version is released.
-*   Merge `master` into the branch for the next release.
-For example, if you are currently on the v1.8 release, you will need to periodically merge `master` into `release-1.8` in order to pick up any changes in `master` during the release process. Therefore, `release-1.8` should effectively be `master` + commits for v1.8.
-*   Update the OSS feature tracking spreadsheet with progress on reviews and merge status for each feature doc, e.g. [Kubernetes Features OSS tracking board (1.7 release)](https://docs.google.com/spreadsheets/d/1IJSTd3MHorwUt8i492GQaKKuAFsZppauT4v1LJ91WHY/edit?usp=sharing).
-*   Attend Kubernetes Tactics meetings.
-*   Attend Kubernetes Burndown meetings.
-*   Sync with the release manager biweekly, then weekly, then daily.
-
-### Work with the engineers and community
-
-*   Sync with the dev managers and SIG leads so that they know to direct the engineers to you if they have any docs questions.
-*   Send announcements to:
-    *   [gke-kubernetes-team@google.com](mailto:gke-kubernetes-team@google.com)
-    *   [kubernetes-sig-release@google.com](mailto:kubernetes-sig-release@google.com), Slack #sig-release
-    *   [kubernetes-sig-leads@google.com](mailto:kubernetes-sig-leads@google.com)
-    *   feature owners listed in the OSS feature tracking spreadsheet
-*   Announcements should cover:
-    *   TL;DR on the docs release process with your contact info.
-    *   Dates for:
-        *   filling in features in the OSS feature tracking spreadsheet
-        *   opening a placeholder PR in the k8s.io repo
-        *   when PRs should be completed and ready for review
-        *   when PRs tech reviews should be completed
-        *   when review feedback should be incorporated by and Docs and Tech LGTMs given
-        *   the drop-dead merge date and time for any last minute PRs
-*   Meet with engineers early about feature docs and give guidance on what should be a Concept, Task, Tutorial, or Reference doc.
-*   Recruit community tech writers to help with docs reviews.
+*   Kubernetes [#sig-release channel](https://kubernetes.slack.com/messages/C2C40FMNF/) on Slack
 
-### Generate reference docs
+## Start the release cycle
+
+Release cycles usually begin 1-2 weeks after the previous cycle completes.
+
+### General starting tasks
+
+1. Get connected.
+
+- Join the Kubernetes [#sig-release Slack channel](https://kubernetes.slack.com/messages/C2C40FMNF/).
```

Comment:
I think that the sub-bullet dashes need to be indented further to align with the 'G' in 'Get connected' to not break up the outer list. Actually I'm not sure if you intended them to be sub-bullets or you just decided to switch from numbered to bullet after the first point. Regardless something is wrong.